### ISSUE TITLE
Attempting to utilize updated libraries in consuming project causes "System.AggregateException"

### ISSUE THREAD

**Issue by Michael-ICS** [harness/ff-dotnet-server-sdk#137]
We have a project that had included references to the following package libraries in our .NET 8 project file.

```
<PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="8.0.2" />
<PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="8.0.2" />
<PackageReference Include="System.Security.Cryptography.Xml" Version="8.0.1" />
```
Having any one of these libraries referenced in our project caused a stack error similar to the following when we attempted to create a CfClient.Instance.

```
System.AggregateException
  HResult=0x80131500
  Message=One or more errors occurred. (Attempt by method 'io.harness.cfsdk.client.connector.HarnessConnector.CreateHttpClientWithTls(io.harness.cfsdk.client.api.Config, Microsoft.Extensions.Logging.ILoggerFactory)' to access method 'Microsoft.IdentityModel.Tokens.CollectionUtilities.IsNullOrEmpty<System.Security.Cryptography.X509Certificates.X509Certificate2>(System.Collections.Generic.IEnumerable`1<System.Security.Cryptography.X509Certificates.X509Certificate2>)' failed.)
  Source=System.Private.CoreLib
  StackTrace:
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Harness.Program.Main(String[] args) in C:\Code\Spike\harness\harness\Program.cs:line 20

  This exception was originally thrown at this call stack:
    [External Code]

Inner Exception 1:
MethodAccessException: Attempt by method 'io.harness.cfsdk.client.connector.HarnessConnector.CreateHttpClientWithTls(io.harness.cfsdk.client.api.Config, Microsoft.Extensions.Logging.ILoggerFactory)' to access method 'Microsoft.IdentityModel.Tokens.CollectionUtilities.IsNullOrEmpty<System.Security.Cryptography.X509Certificates.X509Certificate2>(System.Collections.Generic.IEnumerable`1<System.Security.Cryptography.X509Certificates.X509Certificate2>)' failed.
```

During our investigation we discovered that "ff-dotnet-server-sdk" package is using "System.IdentityModel.Tokens.Jwt (6.34.0). This package's dependencies do not allow for using later versions of the same package. We have business policies to keep our packages, if possible, to the latest available and we are hoping that this may be possible with your library. If your team is willing to update your project or take PRs that would be fantastic.

Example.csproj
```
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net8.0</TargetFramework>
        <Nullable>enable</Nullable>
        <ImplicitUsings>enable</ImplicitUsings>
        <LangVersion>latest</LangVersion>
    </PropertyGroup>

    <ItemGroup>
        <PackageReference Include="Disruptor" Version="6.0.0" />
        <PackageReference Include="ff-dotnet-server-sdk" Version="1.7.1" />
        <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />
        <PackageReference Include="Microsoft.Extensions.Http" Version="8.0.0" />
        <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
        <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
        <PackageReference Include="murmurhash" Version="1.0.3" />
        <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
        <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
        <!-- Comment out the line below to allow the project to run -->
        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="8.0.2" />
    </ItemGroup>

</Project>
```

Program.cs (Based on your example provided from your documentation)
```
using io.harness.cfsdk.client.dto;
using io.harness.cfsdk.client.api;

namespace Harness
{
    internal class Program
    {
        public static String apiKey = Environment.GetEnvironmentVariable("FF_API_KEY");
        public static String flagName = Environment.GetEnvironmentVariable("FF_FLAG_NAME") is string v && v.Length > 0 ? v : "harnessappdemodarkmode";

        static void Main(string[] args)
        {
            // Configure your logger
            var harnessConfig = Config.Builder()
                .SetAnalyticsEnabled(true)
                .Build();

            // Create a feature flag client
            CfClient.Instance.Initialize(apiKey, harnessConfig).Wait();

            var isInit = CfClient.Instance.WaitForInitialization(30000);
            if (!isInit)
            {
                Console.WriteLine("Failed to init the SDK within 30seconds");
            }

            // Create a target (different targets can get different results based on rules)
            Target target = Target.builder()
                .Name("default")
                .Identifier("default")
                .build();

            // Loop forever reporting the state of the flag
            while (true)
            {
                bool resultBool = CfClient.Instance.boolVariation(flagName, target, false);
                Console.WriteLine("Flag variation " + resultBool);
                Thread.Sleep(10 * 1000);
            }
        }
    }
}

```

**Comment by Michael-ICS** [comment]
I'll take a look at the PR notes.

**Comment by erdirowlands** [comment]
Closing as @Michael-ICS 's changes have been released in 1.7.2
